deploy/helm-chart/README.md

```diff
@@ -32,27 +32,34 @@ This chart deploys the NGINX Gateway Fabric in your Kubernetes cluster.
 
 > **Note**
 >
-> The Gateway API resources from the standard channel must be installed
-> before deploying NGINX Gateway Fabric. If they are already installed in your cluster, please ensure they are
-> the correct version as supported by the NGINX Gateway Fabric -
+> The [Gateway API resources](https://github.com/kubernetes-sigs/gateway-api) from the standard channel must be
+> installed before deploying NGINX Gateway Fabric. If they are already installed in your cluster, please ensure
+> they are the correct version as supported by the NGINX Gateway Fabric -
 > [see the Technical Specifications](https://github.com/nginxinc/nginx-gateway-fabric/blob/main/README.md#technical-specifications).
 
-To install the Gateway API CRDs from [the Gateway API repo](https://github.com/kubernetes-sigs/gateway-api), run:
+If installing the latest stable release of NGINX Gateway Fabric, ensure you are deploying v0.8.1 of the
```

Comment:
Do we need to include the version here? It's just another string we will need to update every release.

Reply:
Good call, I can make it more generic.